curriculum/challenges/english/14-responsive-web-design-22/learn-intermediate-css-by-building-a-cat-painting/64a2687ef267e5934a2f93e3.md

```diff
@@ -0,0 +1,218 @@
+---
+id: 64a2687ef267e5934a2f93e3
+title: Step 59
+challengeType: 0
+dashedName: step-59
+---
+
+# --description--
+
+Using a descendant selector, select the two `div` elements inside the `div` with class `cat-mouth`. Give it a width of `30px`, a height of `50px`, and a border of `2px solid #000`.
+
+# --hints--
```

Comment:
Some more variations that would be good to include:

```css
    .cat-mouth > cat-mouth-line-left, .cat-mouth > cat-mouth-line-right {
    
    /* code here */
    
    }
         
    
    .cat-mouth cat-mouth-line-left, .cat-mouth cat-mouth-line-right {
    
      /* code here */
    
    }
    
     
    .cat-mouth > div, .cat-mouth > div {
    
    /* code here */ 
    
    }
    
    
    .cat-mouth div, .cat-mouth div {
    
    /* code here */
    
    }
```

Reply:
THis also works:
```css
.cat-mouth div {
    
    }
```

Especially for the whiskers `div` elements which have three `div` elements inside.

Is that okay by you?